src/ray/core_worker/store_provider/memory_store/memory_store.h

```diff
@@ -174,13 +175,16 @@ class CoreWorkerMemoryStore {
   /// See the public version of `Get` for meaning of the other arguments.
   /// \param[in] abort_if_any_object_is_exception Whether we should abort if any object
   /// resources. is an exception.
+  /// \param[in] abort_after_num_objects Whether we should abort the loop that fills
+  /// results after after we find num_objects. False only for Wait.
```

Comment:
```suggestion
  /// \param[in] at_most_num_objects Whether this function will return *at most* num_objects even if more are ready.
```

Reply:
writing this comment makes the suggested name change even more clear: as written, you are exposing implementation details of the `GetImpl` function rather than documenting the semantics a caller needs to care about